### DISCUSSION TITLE
技术播客月 Next Step

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [opensource-f2f/episode#151] [Polls]
[首次技术播客月](https://github.com/opensource-f2f/episode/issues/96)已经顺利结束，根据[复盘会议](https://www.bilibili.com/video/BV17V4y1773i?spm_id_from=333.999.0.0&vd_source=ca1041974e9b44975cc4d5355ad7001f)上大家的讨论，后续有一些行动项需要完成，包括：

* 创建 GitHub Org 并作为主要的公开协作平台
  * 之前创建的相关 issue 或转移或关联到新的 repo 中
* 创建官网，方便宣传、展示
* @LinuxSuRen （或其他人）联系感兴趣的朋友一起讨论下一届的筹备事项

> 投票选项尚待补充，投票还没开始
下面有个投票需要大家参与：

**Comment by MikeyWei** [comment]
> * 创建官网，方便宣传、展示

是打算为“技术播客月”创建一个官网？

**Reply by LinuxSuRen** [reply]
是的。另外，除了“技术播客月”之外，我们还可以围绕技术类播客做更多的事情，“播客月”可以只是其中的一个活动。

**Comment by LinuxSuRen** [comment]
投票时间截止到本周末（2022-08-21）

**Comment by LinuxSuRen** [comment]
GitHub org 已经创建好了，欢迎加入！
https://github.com/tech-podcasts
